services/movie.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional, List
+
+from django.db.models import QuerySet
+
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids: Optional[List[int]] = None,
+               actors_ids: Optional[List[int]] = None) -> QuerySet[Movie]:
+    movies = Movie.objects.all()
+
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Optional[Movie]:
+    return Movie.objects.filter(id=movie_id).first()
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: Optional[List[int]] = None,
+        actors_ids: Optional[List[int]] = None
+) -> Movie:
+    movie = Movie.objects.create(title=movie_title,
+                                 description=movie_description)
+
+    if genres_ids:
+        genres = Genre.objects.filter(id__in=genres_ids)
+        if genres.exists():
+            movie.genres.add(*genres)
+
+    if actors_ids:
+        actors = Actor.objects.filter(id__in=actors_ids)
+        if actors.exists():
```

Comment:
The check `if actors.exists():` is unnecessary. The `add` method will handle empty querysets gracefully, so you can directly add the actors without checking if they exist.